src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts

```diff
@@ -262,6 +262,30 @@ export class SolutionExplorerList {
     }, 0);
   }
 
+  public getPartToDisplayOfSolutionUri(solutionUri: string): string {
+    const solutionIsRemote: boolean = solutionUri.startsWith('http');
+    if (solutionIsRemote) {
+      return solutionUri;
+    }
+
+    const lastIndexOfSlash: number = solutionUri.lastIndexOf('/');
+    const lastIndexOfBackSlash: number = solutionUri.lastIndexOf('\\');
+    const lastFolderIndex: number = Math.max(lastIndexOfSlash, lastIndexOfBackSlash) + 1;
+
+    const folderName: string = solutionUri.substring(lastFolderIndex);
+
+    const folderNameIsEmpty: boolean = folderName.length === 0;
+    if (folderNameIsEmpty) {
+      return solutionUri;
+    }
+
+    return folderName;
```

Comment:
```suggestion
    const solutionName: string = folderName;
    return solutionName;
```

Either do this. because, sometimes you just want to see the method name and have a quick look at the return.

Reply:
You could also call the `folderName` variable `solutionName` in the first place.

So this would be more a business naming, rather than a technical one.